### ISSUE TITLE
demonstrate model deployment through tracking API (MLflow)

### ISSUE THREAD

**Issue by pascal456** [fhdw-forschung/modelling-tools#82]
# Problem
- Models are already tracked through the Tracking-API (MLflow) in case of the convenience function
- they could be served with PyCarets `make_api`
- this would only apply to models created with PyCaret

# idea
- better solution would be the deployment through MLflow
- This way the deployment would be ML-framework agnostic

# content
- track models into **Stage** and **Production** phases
- demonstrate how to deploy a model through the Tracking-API, for example semi-interactive with notebooks or documentation
- take into account that it might be of interest what instances of the data where used for training; these instances should not be used for demonstration (or explicitly label them as training data); therefore find out how to extract the training instances from the model in the stage phase / find out if this is even possible after transitioning the phase in the registry

# blocked by
- #81

**Comment by pascal456** [comment]
related:
- https://github.com/fhdw-forschung/bebefam-exploration/issues/74

**Comment by pascal456** [comment]
done with #100
